File: 485-max-consecutive-ones/max-consecutive-ones.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        c=0
        r=0
        for i in nums:
            if i==0:
                c=0
            else:
                c=c+1
            r=max(c,r)
        return r
        

File: 485-max-consecutive-ones/max-consecutive-ones.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        int c=0;
        int r=0;
        for(int i :nums){
           if(i==0){
            c=0;
           }else{
            c++;
           }
           r=Math.max(r,c);


        }
        return r;

        
    }
}